Taskfile.yml

```diff
@@ -87,26 +87,26 @@ tasks:
     desc: "Run all integration tests in parallel"
     deps:
       - task: test-integration-db
-        vars: { DB: "FerretDB" }
+        vars: { DB: "FerretDB", LOG: "debug" }
```

Comment:
I would suggest default log level info, I set it all the time. WDYT?

Reply:
I set debug level as I don't want to change the current behavior.